src/main/java/frc/robot/commands/ClimberPosition.java

```diff
@@ -39,26 +28,15 @@ public void initialize() {
 
     if (climber != null )
       climber.setClimberPosition();
-
-    
   }
     
   // Called every time the scheduler runs while the command is scheduled.
   @Override
   public void execute() {
-    if (climber == null){
-      return;
-
-      //System.out.println("auto mode: "+ Robot.m_bAutoMode);
-      // System.out.println("indexer sensor: "+ indexer.m_bForIntakeGoodSensors);
-    }
-    else{
+    if (climber != null) 
       climber.startClimber(14);
```

Comment:
The naming seems confusing, why would you execute a startClimber() command every 20ms?    It would make sense it it was climb() of continueclimb() or something like that.